fil_actor_interface/src/builtin/datacap/mod.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use anyhow::Context;
+use cid::Cid;
+use fvm::state_tree::ActorState;
+use fvm_ipld_blockstore::Blockstore;
+use serde::Serialize;
+
+use crate::io::get_obj;
+
+/// Datacap actor method.
+pub type Method = fil_actor_datacap_v10::Method;
+
+/// Datacap actor state.
+#[derive(Serialize, Debug)]
+#[serde(untagged)]
+pub enum State {
+    V10(fil_actor_datacap_v10::State),
+}
+
+pub fn is_v10_datacap_cid(cid: &Cid) -> bool {
+    let known_cids = vec![
+        // calibnet v10
+        Cid::try_from("bafk2bzaceaot6tv6p4cat3cg5fknq22htosw3p5rwyijmdsraatwqyc4qyero").unwrap(),
+        // mainnet v10
+        Cid::try_from("bafk2bzacealj5uk7wixhvk7l5tnredtelralwnceafqq34nb2lbylhtuyo64u").unwrap(),
+        // devnet v10
+        Cid::try_from("bafk2bzaceagg4qklzhhg5oj4shwqpoeykeyxus7xhj2abuot2tycdwsf2oaaa").unwrap(),
```

Comment:
What is `devnet` precisely in this case?

Reply:
I see we are using `devnet` config for some v10 actors like system, multisig, cron
so I added this for newly added actors

Reply:
`devnet` is custom configuration that Lotus uses for locally run networks.

Reply:
I still don't get it. What is devnet? Where did you get that Cid from? I don't see it in the release https://github.com/filecoin-project/builtin-actors/releases/tag/v10.0.0

Reply:
I got these from https://github.com/filecoin-project/lotus/blob/master/build/builtin_actors_gen.go

Reply:
So why do we need it here? And if we need it, why don't we add all other networks that Lotus supports?

Reply:
We don't need it. I think it was included a long time ago because it got mistaken for the calibration network.

Reply:
then i'll remove all the devnet cid's for all actors

Reply:
Then let's remove it from here and all other occurrences in the repo. I can be a distinct PR if we want to have them separated.